### ISSUE TITLE
No Content response

### ISSUE THREAD

**Issue by Dismissile** [aliencube/AzureFunctions.Extensions#50]
https://swagger.io/docs/specification/describing-responses/

Some responses, such as 204 No Content, have no body. To indicate the response body is empty, do not specify a content for the response:

```yaml
responses:
        '204':
          description: The resource was deleted successfully.
```

If I try and provide `[OpenApiResponseBody]` with `bodyType` of null it throws a `NullReferenceException`.

Need a way to provide a response without content/schema for a No Content response.

**Comment by justinyoo** [comment]
@Dismissile Thanks for the issue! I'll have a look.

**Comment by poroo** [comment]
I stumbled upon the same issue, here's a pull request that allows [OpenApiResponse] -attributes.
https://github.com/aliencube/AzureFunctions.Extensions/pull/60

**Comment by jbroenstrup** [comment]
Yes. This function would be amazing! Also I'd like to leave the body of 404 responses empty.

**Comment by justinyoo** [comment]
It's now available from [2.0.0.1](https://github.com/aliencube/AzureFunctions.Extensions/releases/tag/swagger-2.0.0.1)
